DeploymentExperiences.md

```diff
@@ -0,0 +1,3 @@
+Date: April 13, 2018
+
+Compared to regular website deployment, I find deploying to Heroku to be tedious. While I was able to setup the website alright the app inself refuses to be pushed to heroku. So I have a page with no app. I prefer using other deployment sites for my python apps. Then again it is Friday 13th; I should retry after midnight; Maybe I will have better success. :}
```

Comment:
Heroku can definitely be frustrating. Try to work through the issues as much as you can and if you get stuck, always feel free to reach out!

Reply:
Try looking at your requirements.txt file again - I don't think you have all of your dependencies in there...you might also take a look at your .env file - make sure that is set up properly. 

```
ALLOWED HOSTS = config('ALLOWED_HOSTS', cast=lambda v: [s.strip() for s in v.split(',')])
```